lib/unifex/code_generator/tie_header.ex

```diff
@@ -0,0 +1,32 @@
+defmodule Unifex.CodeGenerator.TieHeader do
+  @moduledoc """
+  Generates connective header file that includes proper header based
+  on selected interface.
+  """
+
+  import Unifex.CodeGenerator.Utils, only: [sigil_g: 2]
+
+  def generate_tie_header(name, interfaces) do
```

Comment:
This function generates header file but with two new lines at the end and I couldn't get rid of one of them. Have no clue what is generating them. It looks like this 
https://github.com/membraneframework/unifex/blob/35e15a77765282961e6680ae8e37f73379ead489/test/fixtures/unified_ref_generated/example.h#L10

Reply:
It's because of triple quotes that 'append' a newline:
```
iex(1)> """
...(1)> abc
...(1)> """
"abc\n"
```

I wouldn't care about that though. `String.trim` may help if it hampers you a lot.